src/screens/Home/index.tsx

```diff
@@ -106,6 +106,9 @@ const Home: React.FC = () => {
 
     const [isInfoHeatOpen, setIsInfoHeatOpen] = useState(false);
 
+    const initialDate = "1/2020";
+    const finalDate = "12/2020";
```

Comment:
These values ​​must not be static, they must be the ones being passed in the request to the backend

Reply:
Disregard this comment, I was wrong